### PR TITLE
Singlepoint training

### PR DESCRIPTION
**Pull request by sunghjung3**
Follow-up pull request from #75 and #82.

The energy/force weight calculations were taken out to a stand-alone function, and the try-except was changed to an if-statement.

Unit tests for single-point training and loss function weights (#93) were also written.

### CODE REVIEW

File: uf3/regression/least_squares.py [hunk 1]
```diff
@@ -1122,3 +1119,28 @@ def find_pair_potential_well(coefficients, rounding_factor):
             # no actual well
             well_idx = peak_idx + 1
     return well_idx
+
+
+def calculate_E_F_weights(n_e, n_f, e_stddev, f_stddev):
```
Comment (JasonGibsonUfl) [inline uf3/regression/least_squares.py:1124]: The naming convention of this method is my only concern. Firstly the suffix `stddev` is too verbose. Please shorten this to `std`. Secondly, please be consistent in your naming `n_e` has the form `Property_variable` while `e_stddev` has the form `variable_property`. 

Lastly, the method name seems long. This is a personal preference but I would suggest the name `calc_E_F_weights`. I will leave that choice to you

**Review by JasonGibsonUfl — COMMENTED** [review]
Looks good. Address the formating comment and I will approve

File: uf3/regression/least_squares.py (hunk 1 above)
Comment (sunghjung3) [reply uf3/regression/least_squares.py]: @JasonGibsonUfl Done

**Review by JasonGibsonUfl — APPROVED** [review]
Looks good

**Review by sxie22 — APPROVED** [review]
